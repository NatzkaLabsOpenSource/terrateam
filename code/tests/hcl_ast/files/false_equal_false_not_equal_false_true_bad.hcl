false = false != false true
---
bar = false != false bar
---
bar = bar != false bar
---
bar = bar != 1 bar
---
bar = bar != "bar" bar
---
bar = bar != 1.3 bar
