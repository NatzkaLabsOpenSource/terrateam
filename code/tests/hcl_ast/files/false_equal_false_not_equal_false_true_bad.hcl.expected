Unexpected literal or identifier found after literal in second operand of not equals operation.
